lambdas/search/index.py

```diff
@@ -202,7 +202,12 @@ def lambda_handler(request):
         }
         size = 0
         _source = False
-        filter_path = 'aggregations.objects.buckets.latest.hits.hits._source'
+        filter_path = ','.join([
+            'took',
+            'timed_out',
+            'hits.total',
+            'aggregations.objects.buckets.latest.hits.hits._source'
+        ])
```

Comment:
Does it make sense to deduplicate these fields here and below?
It looks like you don't have to join these strings:
https://elasticsearch-py.readthedocs.io/en/6.3.1/api.html#response-filtering

(also I'd use a tuple instead of a list, but that's nano-optimization)

Reply:
as for deduping -- i dont think it's worth it